lms/djangoapps/discussion/rest_api/api.py

```diff
@@ -263,6 +275,12 @@ def _format_datetime(dt):
         ),
         "allow_anonymous": course.allow_anonymous,
         "allow_anonymous_to_peers": course.allow_anonymous_to_peers,
+        "user_roles": user_roles,
+        "user_is_privileged": bool(user_roles & {
+            FORUM_ROLE_ADMINISTRATOR,
+            FORUM_ROLE_MODERATOR,
+            FORUM_ROLE_COMMUNITY_TA,
```

Comment:
Just making sure that FORUM_ROLE_GROUP_MODERATOR is not an appropriate role for user to be considered privileged

Reply:
This is for discussions only, in which sense they do have privileges.